granulate_utils/linux/proc_events.py

```diff
@@ -269,7 +269,7 @@ def wrapper(*args, **kwargs):
             if _proc_events_listener is None:
                 try:
                     # needs to run in init net NS - see netlink_kernel_create() call on init_net in cn_init().
-                    _proc_events_listener = ns.run_in_ns(["net"], _start_listener)
+                    _proc_events_listener = ns.run_in_ns_wrapper(["net"], _start_listener)
```

Comment:
proc_events won't work if we're not root, so might as well not fix it and let the PermissionError pop.

In general, let's change only needed sites to `run_in_ns_wrapper` and keep others at `run_in_ns`.